src/app/lib/config/services/arabic.js

```diff
@@ -1,110 +1,114 @@
 import { C_POSTBOX } from '@bbc/psammead-styles/colours';
-import { latin } from '@bbc/gel-foundations/scripts';
+import { arabic } from '@bbc/gel-foundations/scripts';
 import { news as brandSVG } from '@bbc/psammead-assets/svgs';
```

Comment:
Noting that in this implementation, the Brand logo remains `News`. 

This change would pull in the correct Brand SVG from the `@bbc/psammead-assets` package.
```
import { arabic as brandSVG } from '@bbc/psammead-assets/svgs';
```
However, the three SVGs for Arabic, Pashto and Urdu aren't currently in the `@bbc/psammead-assets` package. There is an issue to update the package to include them. https://github.com/bbc/psammead/issues/635 Then in a future Simorgh PR these brand SVGs can be imported in.

Reply:
Yep, spotted that :)